### ISSUE TITLE
Input text is the same as placeholder text

### ISSUE THREAD

**Issue by princefishthrower** [piyushkdas0611/Profile-Card-Generator#37]
I believe that the input fields on the app should be updated to be easier to read, otherwise, even after filling the form fields, they all still look. This is likely breaking the WCAG standard for color contrast on a website: https://webaim.org/resources/contrastchecker/

Only the email field has a high enough contrast!

![Screen Shot 2023-10-18 at 09 30 50](https://github.com/piyushkdas0611/Profile-Card-Generator/assets/11366744/cd4bbd9b-5954-4cf1-b9ca-2edaaf2a71d4)

**Comment by ShivamMaheshwari04** [comment]
@piyushkdas0611 Please Assign this task to me
